src/netif.rs

```diff
@@ -897,18 +901,11 @@ mod driver {
         ///   `EspNetif` instance is expected to be a PPP netif.
         /// * The `tx` callback implementation is simply expected to write the
         ///   PPP packet into e.g. UART
-        #[cfg(esp_idf_lwip_ppp_support)]
```

Comment:
did you accidentally removed the ppp_support cfg here?

Reply:
Yes but this method will be removed shortly, as it no longer makes sense. New changes upcoming. Will temporarily turn into "draft".